audit_exceptions/versioned_dependencies_conflicts_allowlist.json

```diff
@@ -1,25 +1,12 @@
 [
-  "apache-arrow-glib",
-  "aqbanking",
   "anime-downloader",
-  "arxiv_latex_cleaner",
-  "aws-google-auth",
   "chapel",
-  "coin3d",
   "emscripten",
-  "freedink",
   "hive",
   "jupyterlab",
   "libgaiagraphics",
   "librasterlite",
-  "mpv",
-  "openmodelica",
   "predictionio",
-  "pyqt",
-  "pyside",
-  "pyside@2",
-  "qca",
   "sqoop",
-  "swtpm",
-  "synergy-core"
+  "swtpm"
```

Comment:
Remaining ones:
- `anime-downloader`, `emscripten`, `jupyterlab` - Due to `node --> python`
- `chapel` - Need to migrate with `python` alias
- `libgaiagraphics`, `librasterlite` - Due to deprecation in favor of `librasterlite2`
- `swtpm` - Need to continue `openssl` migration
- `hive`, `predictionio`, `sqoop` - Existing entries either deprecated or pending upstream support